std_msgs/Header header
int32 xian_dj_retractable_platform_stand_linear_actuator_stand_cmd 
int32 xian_dj_retractable_platform_stand_linear_actuator_sit_cmd 
int32 xian_dj_retractable_platform_first_linear_actuator_up_cmd 
int32 xian_dj_retractable_platform_first_linear_actuator_down_cmd 
int32 xian_dj_retractable_platform_second_linear_actuator_up_cmd 
int32 xian_dj_retractable_platform_second_linear_actuator_down_cmd 